model_compression_toolkit/qat/pytorch/quantization_facade.py

```diff
@@ -181,6 +183,7 @@ def pytorch_quantization_aware_training_init_experimental(in_model: Module,
 
         user_info.mixed_precision_cfg = bit_widths_config
 
+        qat_model.graph = qat_model.graph.get_internal_graph()
```

Comment:
why does the returned model contain the fused graph and not the original?
I'm not sure that I understand what is going on in back2fw and here, and also, this is kind of criptyc (sort of a hidden assumption that can cause problems)